core/templates/dev/head/pages/exploration_editor/statistics_tab/PlaythroughIssuesService.js

```diff
@@ -79,9 +80,16 @@ oppia.factory('PlaythroughIssuesService', [
     };
 
     return {
-      initSession: function(newExplorationId, newExplorationVersion) {
+      initSession: function(
+          newExplorationId, newExplorationVersion,
+          newWhitelistedExplorationIds) {
         explorationId = newExplorationId;
         explorationVersion = newExplorationVersion;
+        whitelistedExplorationIds = newWhitelistedExplorationIds || [];
```

Comment:
Hi @brianrodri, I am a bit confused about this. Why do we need the `|| []`? In other words why is newWhitelistedExplorationIds ever null?